### PR TITLE
Km/left nav tests

### PR DESCRIPTION
**Pull request by katiemcculloch**

### CODE REVIEW

File: cypress/integration/utils/sharedTests/emptyRepo.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import {
```
Comment (katiemcculloch) [inline cypress/integration/utils/sharedTests/emptyRepo.ts:1]: Ill be adding more empty repo tests to this file

File: cypress/integration/utils/sharedTests/repoDatabaseNav.ts [hunk 2]
```diff
@@ -0,0 +1,286 @@
+import {
```
Comment (katiemcculloch) [inline cypress/integration/utils/sharedTests/repoDatabaseNav.ts:1]: I copied all these tests over from `sharedTests/repoDatabaseNav` (eventually I'll delete `repoLeftNav`).

The only thing I added was the`clickToOpenNavClickFlow` function and used it to open navbar before running assertions. 

Im going to clean up this file later, like move query expectations to the query page, and so on.

File: cypress/integration/productionTests/publicPaths/render/repository/emptyWithBranch.spec.ts [hunk 3]
```diff
@@ -1,16 +1,13 @@
+import { testDoltInstallationSteps } from "cypress/integration/utils/sharedTests/emptyRepo";
+import { testRepoHeaderWithBranch } from "cypress/integration/utils/sharedTests/repoHeaderNav";
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/repository/emptyWithBranch.spec.ts:2]: These imports shouldn't be absolute...in a few of these files

**Review by tbantle22 — APPROVED** [review]
One comment, otherwise LGTM

File: cypress/integration/productionTests/publicPaths/render/repository/emptyWithBranch.spec.ts (hunk 3 above)
Comment (katiemcculloch) [reply cypress/integration/productionTests/publicPaths/render/repository/emptyWithBranch.spec.ts:2]: whooooops that damn auto import fixer......... thanks, will fix

Comment (katiemcculloch) [reply cypress/integration/productionTests/publicPaths/render/repository/emptyWithBranch.spec.ts]: https://github.com/dolthub/dolthub-cypress/pull/140/commits/26c85b752fa16d098f3f0690430c5c439bc2fd0e

**Pull request merged**